specification/network/resource-manager/Microsoft.Network/stable/2022-07-01/networkVirtualAppliance.json

```diff
@@ -1170,6 +1186,44 @@
         }
       },
       "description": "Properties of the Inbound Security Rules resource."
+    },
+    "DelegationProperties": {
+      "type": "object",
+      "properties": {
+        "serviceName": {
+          "type": "string",
+          "description": "The service name to which nva is delegated."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState"
+        }
+      },
+      "description": "Properties of the delegation."
+    },
+    "PartnerManagedResourceProperties": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "format": "arm-id",
+          "description": "The partner managed resource id."
+        },
+        "internalLoadBalancerId": {
+          "readOnly": true,
+          "type": "string",
+          "format": "arm-id",
+          "description": "The partner managed ILB resource id"
+        },
+        "standardLoadBalancerId": {
+          "readOnly": true,
+          "type": "string",
+          "format": "arm-id",
+          "description": "The partner managed SLB resource id"
+        }
```

Comment:
Looks like you are supposed to set the x-ms-arm-id-details also to specify that they are load balancer resource types?

Reply:
Address the review comments. 
Specified the respective resource types